Makefile

```diff
@@ -28,23 +28,26 @@ CMDS = .
 build: | $(O) ## Build evy binaries
 	go build -o $(O) -ldflags='$(GO_LDFLAGS)' $(CMDS)
 
-install: ## Build and install binaries in $GOBIN
+## Build and install binaries in $GOBIN
+install:
 	go install -ldflags='$(GO_LDFLAGS)' $(CMDS)
 
 # Use `go version` to ensure the right go version is installed when using tinygo.
 go-version:
 	go version
 
-# Optimise tinygo output for size, see https://www.fermyon.com/blog/optimizing-tinygo-wasm
-tiny: go-version | $(O) ## Build for tinygo / wasm
+## Build for tinygo / wasm for size, see https://www.fermyon.com/blog/optimizing-tinygo-wasm
```

Comment:
Should that `see https://...` stuff really be part of the help message? If not, you can put it on the next line with a single `#` comment marker. or you can put it on the line before - again, single comment marker.

Reply:
done.